plutus-tx/src/PlutusTx/Code.hs

```diff
@@ -25,6 +25,7 @@ import           Flat.Decoder            (DecodeException)
 import qualified Data.ByteString         as BS
 import qualified Data.ByteString.Lazy    as BSL
 import           ErrorCode
+import           Prelude
```

Comment:
Perhaps a comment above the import in the modules where we do this. e.g. this one is exclusively off-chain code, so we can do this.